test/verify/check-users

```diff
@@ -761,6 +761,7 @@ class TestAccounts(MachineCase):
         b.assert_pixels("#group-confirm-delete-dialog", "group-delete-dialog")
         b.click("#group-confirm-delete-dialog footer .pf-c-button.apply")
         b.wait_not_present('#account-confirm-delete-dialog')
+        b.wait(lambda: "#/" == b.eval_js('window.location.hash'))
```

Comment:
Why is this necessary? Isn't `b.wait_visible("#groups-list")` sufficient enough?

Reply:
It should be sufficient but since we noticed that there is this race with setting the url, I thought I would guard it in a stricter way.